src/net_processing.cpp

```diff
@@ -2179,9 +2250,6 @@ bool static ProcessMessage(CNode* pfrom, const std::string& strCommand, CDataStr
             AcceptToMemoryPool(mempool, state, ptx, &fMissingInputs, &lRemovedTxn, false /* bypass_limits */, 0 /* nAbsurdFee */)) {
```

Comment:
Looks like `lRemovedTxn` still needs to be cleared in this method, unless moving it into `ProcessOrphans` is appropriate.

Reply:
Sorry, that indeed looks ugly right now. I will rebase this on #11775, which removes `lRemovedTxn` in whole.